.github/workflows/potential-duplicates.yml

```diff
@@ -0,0 +1,36 @@
+name: Potential Duplicates
+on:
+  issues:
+    types: [opened, edited]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: bubkoo/potential-duplicates@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # Label to set, when potential duplicates are detected.
+          label: potential duplicate
+          # Exclude keyworlds in title before detecting.
+          exclude: |
+            icon
+            icons
+            logo
+            logos
+            request
+            request:
+            add
+            adding
+          # Get issues with state to compare. Supported state: 'all', 'closed', 'open'.
+          state: all
+          # If similarity is higher than this threshold([0,1]), issue will be marked as duplicate.
+          threshold: 0.7
+          # Reactions to be add to comment when potential duplicates are detected.
+          # Available reactions: "-1", "+1", "confused", "laugh", "heart", "hooray", "rocket", "eyes"
+          reactions: eyes
```

Comment:
Personally not so much of a fan of this, what does it add that the label doesn't? :thinking:

Reply:
Totally agree, I kept it there to test it out, but it does not add any real value. Will remove it!